TrackingTools/TrackAssociator/src/TrackDetectorAssociator.cc

```diff
@@ -804,9 +804,9 @@ void TrackDetectorAssociator::fillMuon( const edm::Event& iEvent,
                  }
               }
 	}
-	// GEM Chamber   
-	else if (parameters.useGEM){
-	  if(const GEMSuperChamber* chamber = dynamic_cast<const GEMSuperChamber*>(geomDet) ) {	 
+	// GEM Chamber or ME0 Chamber
+	else if (parameters.useGEM || parameters.useME0){
```

Comment:
looks like checks for values of useGEM and useME0 should be added to individual GEM and ME0 chamber ifs